.github/workflows/build-test-windows-x86.yml

```diff
@@ -378,6 +378,8 @@ jobs:
           cp d:/cygwin/usr/x86_64-w64-mingw32/sys-root/mingw/bin/zlib1.dll artifacts/
           cp d:/cygwin/usr/x86_64-w64-mingw32/sys-root/mingw/bin/iconv.dll artifacts/
           cp d:/cygwin/usr/x86_64-w64-mingw32/sys-root/mingw/bin/libintl-8.dll artifacts/
+          cp "c:/Program Files/Amazon/AWSCLIV2/libcrypto-3.dll" artifacts/
+          cp "c:/Program Files/Amazon/AWSCLIV2/libssl-3.dll" artifacts/
```

Comment:
This is pretty fragile, since it depends on the GHA image configuration...
We need something more reproducible...

Reply:
But it will unblock our next release.